packages/xrpl/test/client/autofill.test.ts

```diff
@@ -28,25 +36,31 @@ describe('client.autofill', function () {
       Sequence,
       LastLedgerSequence,
     }
-    const txResult = await this.client.autofill(tx)
+    const txResult = await testContext.client.autofill(tx)
 
     assert.strictEqual(txResult.Fee, Fee)
     assert.strictEqual(txResult.Sequence, Sequence)
     assert.strictEqual(txResult.LastLedgerSequence, LastLedgerSequence)
   })
 
-  it('converts Account & Destination X-address to their classic address', async function () {
+  it('converts Account & Destination X-address to their classic address', async () => {
     const tx: Payment = {
       TransactionType: 'Payment',
       Account: 'XVLhHMPHU98es4dbozjVtdWzVrDjtV18pX8yuPT7y4xaEHi',
       Amount: '1234',
       Destination: 'X7AcgcsBL6XDcUb289X4mJ8djcdyKaB5hJDWMArnXr61cqZ',
     }
-    this.mockRippled.addResponse('account_info', rippled.account_info.normal)
-    this.mockRippled.addResponse('server_info', rippled.server_info.normal)
-    this.mockRippled.addResponse('ledger', rippled.ledger.normal)
+    testContext.mockRippled!.addResponse(
```

Comment:
Why does this need the `!`? Isn't it guaranteed to exist?

Reply:
The context object is shared between tests and the mockRippled doesn't always exist between `setupMockRippledConnection` and `setupMockRippledConnectionForBroadcast`